lib/active_fedora/associations/has_and_belongs_to_many_association.rb

```diff
@@ -96,6 +82,12 @@ def stale_state
           owner[reflection.foreign_key]
         end
 
+        def find_target
+          ids = owner[reflection.foreign_key]
+          return [] if ids.blank?
+          ActiveFedora::Base.find(ids)
```

Comment:
@mjgiarlo no, it's still querying Solr, it's just doing it using the easily understandable `find` method.